spec/models/bulk_submission_spec.rb

```diff
@@ -297,8 +297,8 @@
       it 'raises an error and sets an error message' do
         expect { subject.process }.to raise_error(ActiveRecord::RecordInvalid)
         expect(subject.errors.messages[:spreadsheet][0]).to eq(
-          "Inconsistent values for column 'scRNA Core Number of Samples per Pool' for " \
-            "Study name 'abc123_study', all rows for a specific study must have the same value"
+          "Inconsistent values for column 'scRNA Core Number of Samples per Pool' for Study name 'abc123_study' " \
+            "and Project name 'Test project', all rows for a specific study and project must have the same value"
```

Comment:
You could add a new test to check the new behaviour - that it DOES allow different values for different projects within the same study.

Reply:
I have [added a line in the valid sample sheet](https://github.com/sanger/sequencescape/pull/4437/commits/87e277e2286ef164cb9196ae4db889015b1e5179#diff-7f888b0ce579bd677f3112b85a86f2715447be6ad073940ac5ce925d93fdbdc1) to cover this case.